### PR TITLE
MB-14446: Add W-2 info to review screen

### PR DESCRIPTION
**Pull request by ronaktruss**
## Summary

This adds the W-2 address info to the review page. I also updated our dev seed data to include filled out w-2 addresses for those that are passed the about page in the flow.

## Setup to Run Your Code

<details>
<summary>💻 You will need to use two separate terminals to test this locally.</summary>

##### Terminal 1

Start the UI locally.

```sh
make client_run
```

##### Terminal 2

Start the Go server locally.

```sh
make server_run
```

</details>

### Additional steps

1. Access the customer app
2. Login as a `weightTicketPPM@ppm.approved `
3. Click the `Upload PPM Documents` button

## Verification Steps for Author

These are to be checked by the author.

- [ ] Tested in the Experimental environment (for changes to containers, app startup, or connection to data stores)
- [ ] Request review from a member of a different team.
- [ ] Have the Jira acceptance criteria been met for this change?

## Verification Steps for Reviewers

These are to be checked by a reviewer.

### Frontend

- [ ] User facing changes have been reviewed by design.
- [ ] There are no aXe warnings for UI.
- [ ] This works in [Supported Browsers and their phone views](https://github.com/transcom/mymove/tree/main/docs/adr/0016-Browser-Support.md) (Chrome, Firefox, IE, Edge).
- [ ] There are no new console errors in the browser devtools
- [ ] There are no new console errors in the test output
- [ ] If this PR adds a new component to Storybook, it ensures the component is fully responsive, OR if it is intentionally not, a wrapping div using the `officeApp` class or custom `min-width` styling is used to hide any states the would not be visible to the user.

## Screenshots
image

### CODE REVIEW

File: src/utils/ppmCloseout.jsx [hunk 1]
```diff
@@ -25,6 +25,31 @@ export const formatAboutYourPPMItem = (ppmShipment, editPath, editParams) => {
...
+            ? `${ppmShipment.w2Address.streetAddress1} ${ppmShipment.w2Address.streetAddress2}`
+            : ppmShipment.w2Address?.streetAddress1,
+          hideLabel: true,
+        },
+        {
+          id: 'addressLine2',
+          label: 'address line2:',
+          value: `${ppmShipment.w2Address?.city}, ${ppmShipment.w2Address?.state} ${ppmShipment.w2Address?.postalCode}`,
+          hideLabel: true,
+        },
```
Comment (ronaktruss) [inline src/utils/ppmCloseout.jsx:52]: Optional chaining is used here to account for the scenarios where a ppm is created past the about page but doesn't have a W-2 address. If we are okay with showing the [something went wrong page](https://storybook.dp3.us/?path=/story/components-something-went-wrong--component) we can get rid of those.

File: src/utils/ppmCloseout.jsx [hunk 2]
```diff
@@ -5,6 +5,21 @@ import moment from 'moment';
 import { formatCents, formatCentsTruncateWhole, formatCustomerDate, formatWeight } from 'utils/formatters';
 import { expenseTypeLabels, expenseTypes } from 'constants/ppmExpenseTypes';
 
+const getW2Address = (address) => {
+  const addressLine1 = address?.streetAddress2
+    ? `${address.streetAddress1} ${address.streetAddress2}`
+    : address?.streetAddress1;
+  const addressLine2 = `${address?.city}, ${address?.state} ${address?.postalCode}`;
```
Comment (ronaktruss) [inline src/utils/ppmCloseout.jsx:12]: See https://github.com/transcom/mymove/pull/9605#discussion_r1024510759
Duplicated below for convenience:
> Optional chaining is used here to account for the scenarios where a ppm has already been created in the past and progressed past the about page before the W-2 address was added. If we are okay with showing the [something went wrong page](https://storybook.dp3.us/?path=/story/components-something-went-wrong--component) we can get rid of these optional chaining operators.

**Comment by rileyorr** [comment]
@ronaktruss the styling LGTM! 

Something I noticed when I edited the W-2 address is that the 'Save & Continue' button redirects to the 'Weight tickets' screen rather than back to the 'Review' page. I then added some fake weight tickets and was brought back to the 'Review' page. When I edited the address a second time, I was once again redirected to the 'Weight tickets' page. Since I'd already uploaded my weight tickets, I used the back button to return to the 'Review' page. Once I did that, the weight ticket information was overwritten with empty values. Unsure if this redirecting issue will be resolved in a separate ticket or if it's a bug. 

Screen recording for reference: 

https://user-images.githubusercontent.com/46329147/202489268-158be750-6a8c-4528-8287-734130af4738.mov

**Comment by ronaktruss** [comment]
> @ronaktruss the styling LGTM!
> 
> Something I noticed when I edited the W-2 address is that the 'Save & Continue' button redirects to the 'Weight tickets' screen rather than back to the 'Review' page. I then added some fake weight tickets and was brought back to the 'Review' page. When I edited the address a second time, I was once again redirected to the 'Weight tickets' page. Since I'd already uploaded my weight tickets, I used the back button to return to the 'Review' page. Once I did that, the weight ticket information was overwritten with empty values. Unsure if this redirecting issue will be resolved in a separate ticket or if it's a bug.
> 
> Screen recording for reference:
> 
>  2022-11-17.PPM.edit.mov

@rileyorr Thanks for reviewing and catching that! This is an existing bug: https://dp3.atlassian.net/browse/MB-13528. This appears to be something that team moose will be working on based on the comments in that ticket.

**Review by rileyorr — APPROVED** [review]
@ronaktruss thank you for linking that bug ticket! I reviewed the Happo diffs so this looks good to go.

File: src/utils/ppmCloseout.jsx (hunk 1 above)
Comment (bellex3) [reply src/utils/ppmCloseout.jsx]: I think having this chaining should be ok bc moving forward, the customer will need to enter their w2 address when filling out the 'about ppm' form. Thus the issue mentioned above ideally shouldn't  happen anymore.

**Review by bellex3 — APPROVED** [review]
@ronaktruss  Looks good to me. I tested the w2 address portion by editing the addresses,  and was able to see updated addresses in the review screen (both with and without address line 2).

**Pull request merged**